pkg/manager/volumes/phase.go

```diff
@@ -66,7 +71,9 @@ func (p VolumePhase) String() string {
 
 func (p *podVolModifier) getVolumePhase(vol *ActualVolume) VolumePhase {
 	if err := p.validate(vol); err != nil {
-		klog.Warningf("volume %s/%s modification is not allowed: %v", vol.PVC.Namespace, vol.PVC.Name, err)
+		if !errors.Is(err, ErrChangeDefaultStorageClass) {
```

Comment:
Increased verbosity: PD defaults to the default class, and it will report this useless warning all the time.